docs/notebooks/06_differential_expression.ipynb

```diff
@@ -0,0 +1,1306 @@
+{
```

Comment:
<p>something seems still wrong here as the plot doesn't show the gene names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>